website/source/assets/javascripts/configuration-builder.js

```diff
@@ -0,0 +1,133 @@
+function downloadConfiguration() {
+  var form = document.querySelector("#configuration-builder");
+  var config = "";
+
+  // Add Listener stanza
+  if (document.getElementById("include_tcp_listener").checked) {
+    config += `listener "tcp" {
```

Comment:
I don't know enough about the compilation here, but does middleman transpile this to ES5? Or will this only be supported in browsers that support ES6 template strings?

Reply:
This middleman setup does not transpile es6, so we need to have all es5 javascript, unfortunately

Reply:
It does, yeah. I thought I remembered IE11 support at least but looks like it might be a good idea to do it the old way
https://caniuse.com/#feat=template-literals

Reply:
(looks like it's just IE that doesn't have native support for this fwiw)

Reply:
Lol I didn't refresh and both of you had already replied 😂 sorry bout that.